src/en/developer-guide/applications/utilities/create-and-open.md

```diff
@@ -0,0 +1,28 @@
+---
+author: Lukáš Gallovič
+---
+
+# Create And Open - Entity Generator
+
+### This is a tutorial on how to use custom TS code to generate an entity from a different entity.
+You have to add `JS function` to RibbonDiff.xml, so it looks something like this.
+You must use `createAndOpen` function and `talxis_entitygenerator.js` library.
+```
+<JavaScriptFunction FunctionName="createAndOpen" Library="$webresource:talxis_entitygenerator.js">
+<CrmParameter Value="PrimaryControl" />
+<StringParameter Value="ait_GenerateCase" /> (name of my action)
+</JavaScriptFunction>
+```
+U must use `2 parameters` the first one is always PrimaryControl and the second one is the name of your workflow action.
+
+Then you have to create an action with the given name. And add 2 Process Arguments just like in the picture
+![Process Arguments](/.attachments/CreateAndOpen.png)
+The names `must` be the same as on the picture.
```

Comment:
you are using '`' character for inline code sample, but I feel like there should be double asterisk for bold instead? '**'

Reply:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code